go.mod

```diff
@@ -20,16 +20,14 @@ require (
 	k8s.io/klog v1.0.0
 	k8s.io/kubernetes v1.14.2
 	k8s.io/utils v0.0.0-20190809000727-6c36bc71fc4a
-	sigs.k8s.io/cluster-api v0.2.4
-	sigs.k8s.io/cluster-api-bootstrap-provider-kubeadm v0.1.2
-	sigs.k8s.io/controller-runtime v0.2.2
+	sigs.k8s.io/cluster-api v0.2.5
+	sigs.k8s.io/cluster-api-bootstrap-provider-kubeadm v0.1.3
+	sigs.k8s.io/controller-runtime v0.3.0
 	sigs.k8s.io/yaml v1.1.0
 )
 
 replace (
 	k8s.io/api => k8s.io/api v0.0.0-20190704095032-f4ca3d3bdf1d
 	k8s.io/apimachinery => k8s.io/apimachinery v0.0.0-20190817020851-f2f3a405f61d
 	k8s.io/client-go => k8s.io/client-go v0.0.0-20190918200256-06eb1244587a
```

Comment:
You should be able to remove these?

Reply:
Although, you'll need to set the correct versions in the require directive

Reply:
Can we, I thought we needed a newer version of client-go for the conversions. @liztio is that still the case?

Reply:
controller-runtime 0.3 has the correct client-go version

Reply:
if you take out the replace, make sure you can pass all tests, build the manager, and run `make bin/clusterctl`